src/components/layouts/Card/StudyCard.tsx

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
+import CountWithIcon from '@ferlab/ui/core/dist/components/labels/CountWithIcon';
 
 import DonorIcon from 'components/interface/Icon/Donor';
 import ExternalLink from 'components/interface/Icon/ExternalLink';
 import FileIcon from 'components/interface/Icon/File';
 import CardContainerNotched from 'components/layouts/Card/CardContainerNotched';
 import CardContent from 'components/layouts/Card/CardContent';
-import CountWithIcon from 'components/layouts/CountWithIcon';
-import StackLayout from 'components/layouts/StackLayout';
 import { t } from 'locales/translate';
 
+import { labelDisplayName } from '../../../utils/labelDisplayName';
```

Comment:
No need to have that component use t from 'locales/translate'. it already does all the work for you

changed it everywhere

Reply:
in the project you can also import all folders "absolute" after src exemple '../../../utils/labelDisplayName' -> 'utils/labelDisplayName'